### PR TITLE
Vaurca Integrated Magclaws

### PR DESCRIPTION
**Pull request by RustingWithYou**
This adds a loadout augment for a set of deployable magboots, available to Vaurca. 

Reasoning: Magboots look dumb on Vaurca, especially Bulwarks, which don't even have sprites for them. Instead of adding the Vaurca-specific magboots all over the place, now the funny bugs can get them in loadout.

I can job-restrict this if required, but I don't feel that magboots are rare enough to be particularly unbalanced.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Review by Arrow768 — CHANGES_REQUESTED** [review]
This needs to be job restricted to engineering/mining.

**Comment by RustingWithYou** [comment]
> This needs to be job restricted to engineering/mining.

Job restrictions have been added

**Comment by NonQueueingMatt** [comment]
!review

File: code/game/objects/items/weapons/vaurca_items.dm [hunk 1]
```diff
@@ -263,13 +263,34 @@
...
+	action_button_name = null
+	item_flags = THICKMATERIAL|AIRTIGHT|INJECTIONPORT|NOSLIP
+	canremove = FALSE
+
+/obj/item/clothing/shoes/magboots/vaurca/aug/throw_at()
+	usr.drop_from_inventory(src)
+
+/obj/item/clothing/shoes/magboots/vaurca/aug/dropped()
+	. = ..()
+	loc = null
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/vaurca_items.dm:288]: this is already handled by qdel

File: code/game/objects/items/weapons/vaurca_items.dm [hunk 2]
```diff
@@ -263,13 +263,34 @@
...
+	qdel(src)
+
+/obj/item/clothing/shoes/magboots/vaurca/aug/negates_gravity()
+	return 1
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/vaurca_items.dm:292]:
```suggestion
	return TRUE
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

File: code/game/objects/items/weapons/vaurca_items.dm [hunk 3]
```diff
@@ -263,13 +263,33 @@
 	contained_sprite = 1
 	icon = 'icons/obj/vaurca_items.dmi'
 
-	species_restricted = list(BODYTYPE_VAURCA,BODYTYPE_VAURCA_WARFORM)
+	species_restricted = list(BODYTYPE_VAURCA,BODYTYPE_VAURCA_WARFORM, BODYTYPE_VAURCA_BULWARK)
```
Comment (FluffyGhoster) [inline code/game/objects/items/weapons/vaurca_items.dm:266]:
```suggestion
	species_restricted = list(BODYTYPE_VAURCA, BODYTYPE_VAURCA_WARFORM, BODYTYPE_VAURCA_BULWARK)
```

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**